### PR TITLE
Ckpt tool

### PR DESCRIPTION
**Pull request by leixiaoning**

### CODE REVIEW

File: athena/utils/checkpoint.py [hunk 1]
```diff
@@ -36,56 +35,42 @@ class Checkpoint(tf.train.CheckpointManager):
...
-        self.best_checkpoint_directory = os.path.join(self.checkpoint_directory, 'best_loss')
+        # load from checkpoint if previous models exist in checkpoint dir
+        self.restore(tf.train.latest_checkpoint(checkpoint_directory))
 
     def _compare_and_save_best(self, loss, save_path):
         """ compare and save the best model in best_loss """
         if loss is None:
             return
         if loss < self.best_loss:
-            self.best_loss = loss
```
Comment (Some-random) [inline athena/utils/checkpoint.py]: we still need to update self.best_loss when a new loss is smaller
